src/streams/EditStreamCard.js

```diff
@@ -26,6 +31,151 @@ type Props = $ReadOnly<{|
+                          return {
+                            text:
+                              role === Role.Admin || role === Role.Owner
+                                ? '{realmName} does not allow anybody to make web-public streams. To change this setting, please open {realmUrl} in your browser.'
+                                : '{realmName} does not allow anybody to make web-public streams.',
+                            values: { realmName, realmUrl: realmUrl.toString() },
+                          };
+                        case 7: // CreateWebPublicStreamPolicy.OwnerOnly
+                          return {
+                            text:
+                              role === Role.Admin
+                                ? '{realmName} only allows organization owners to make web-public streams. To change this setting, please open {realmUrl} in your browser.'
+                                : '{realmName} only allows organization owners to make web-public streams.',
+                            values: { realmName, realmUrl: realmUrl.toString() },
+                          };
+                        case 2: // CreateWebPublicStreamPolicy.AdminOrAbove
+                          return {
+                            text:
+                              '{realmName} only allows organization admins or owners to make web-public streams.',
+                            values: { realmName },
+                          };
+                        case 4: // CreateWebPublicStreamPolicy.ModeratorOrAbove
+                          return {
+                            text:
+                              '{realmName} only allows organization moderators, admins, or owners to make web-public streams.',
+                            values: { realmName },
+                          };
+                        default: {
+                          ensureUnreachable(createWebPublicStreamPolicy);
+
+                          // (Unreachable as long as the cases are exhaustive.)
+                          return '';
+                        }
+                      }
+                    })(),
+                    learnMoreUrl:
+                      role === Role.Admin || role === Role.Owner
+                        ? new URL('/help/configure-who-can-create-streams', realmUrl)
+                        : undefined,
+                  }
+                : false,
+            }
+          : undefined,
+        {
+          key: 'public',
+          title: 'Public',
+          subtitle:
+            'Organization members can join (guests must be invited by a subscriber); organization members can view complete message history without joining',
+        },
+        {
+          key: 'invite-only-public-history',
+          title: 'Private, shared history',
+          subtitle:
+            'Must be invited by a subscriber; new subscribers can view complete message history; hidden from non-administrator users',
```

Comment:
These strings are meant to match what's on web, right? Probably a comment somewhere here briefly pointing at the corresponding web code for comparison would be helpful.